Tutoring services document

Hamled opened the pull request:
This is the document that we discussed creating at the instructional staff meeting on 2/22.

I tried to make clear the differences in our tutoring services, especially in how easy it would be to get tutoring through one service versus another. I chose the term Ad-hoc Tutoring for when students arrange tutoring themselves through the mailing list, let me know if you have a better term for that!

I also included a short FAQ item that lays out very explicitly that students should first try Open Tutoring and Ad-hoc Tutoring before talking with their instructor about setting up Scheduled Tutoring.

Hamled committed:
Tutoring services document

kariabancroft commented on tutoring.md:
```diff
@@ -0,0 +1,57 @@
+1. [Services](#services)
+  1. [Open Tutoring](#open-tutoring)
+  1. [Ad-hoc Tutoring](#ad-hoc-tutoring)
+  1. [Secheduled Tutoring](#scheduled-tutoring)
```
misspelled

kariabancroft commented on tutoring.md:
```diff
@@ -0,0 +1,57 @@
+## Overview
+Ada Developers Academy is committed to ensuring that all of our students receive the support they need to succeed with our curriculum and go on to become professional software developers.
+
+To that end we offer a number avenues for accessing tutoring services outside of class hours. This document explains the different tutoring services offered including when and where they are available and has a FAQ for guiding you through the process of selecting an appropriate tutoring service.
```
"of avenues"

kariabancroft commented on tutoring.md:
```diff
@@ -0,0 +1,57 @@
+### Open Tutoring
+Open Tutoring is our most available tutoring service and should likely be your first choice if you feel tutoring is necessary.
+
+Open Tutoring consists of a fixed set of hours during which experienced software developers are available at the Ada campus to answer questions and assist with debugging for any students present at that time.
```
lol @ "campus"

CheezItMan commented on tutoring.md:
```diff
@@ -0,0 +1,57 @@
+
+(Last updated 2/23/2016 - Check your class Google calendar for temporary schedule changes.)
+
+### Ad-hoc Tutoring
```
Maybe a link to the google group.  Someone's likely to Google it and find the Google group for the Ada language.

sudocrystal replied:
I think linking to the email address is better (ada-tutors@googlegroups.com) but if you want the forum link, this is it: https://groups.google.com/forum/#!forum/ada-tutors

sudocrystal replied:
Oops - just realized that my above comment doesn't make sense in this context.

sudocrystal replied:
Also - just FYI Brooks will end his sessions at the end of Ruby - so that's pretty soon. Maybe we can see if someone else with Rails knowledge (maybe a previous Adie) would be willing to do a Saturday session?

Hamled replied:
I didn't want to include the actual email address because of automated spam bots picking it up. The forum link won't help because it's only for people who are actually members of the list (which is just the tutors and staff).

Maybe that's not such a big problem, though. Also the email address is now listed on this pull request so it's out there either way.

kariabancroft commented on tutoring.md:
```diff
@@ -0,0 +1,57 @@
+- What times would work best for you
+- Something about yourself and/or your preferred learning and communication styles
+
+Once you've posted a message to the mailing list an Ada staff member will follow up with you within a few days to make sure that you were able to find assistance or help to make other arrangements.
```
Isn't it an e-mail? "posted a message" seems like it's through the google groups forum.

sudocrystal replied:
Yes it's an email to ada-tutors@googlegroups.com

sudocrystal replied:
Should we mention here that ad-hoc tutoring can be used to dive deeper into a concept, not just for remediation (different word choice would be better here but can't think at the moment)

Hamled replied:
I can use the terminology "send an email" instead.

Hamled replied:
@sudocrystal I can try to include some wording about going into more depth on particular topics.

kariabancroft commented on tutoring.md:
```diff
@@ -0,0 +1,57 @@
+Once you've posted a message to the mailing list an Ada staff member will follow up with you within a few days to make sure that you were able to find assistance or help to make other arrangements.
+
+### Scheduled Tutoring
+Scheduled Tutoring is generally reserved for students who have need of recurrent, dedicated instruction outside of class such as when significant instruction material has been missed due to absences.
```
"who are in need of"

kariabancroft replied:
I don't think the "absence" example is good here since that's not the case for 99% of our scheduled tutoring folks. We were actually talking about the ad-hoc tutoring being for extended absences

sudocrystal replied:
Yeah I agree - absences generally lead to ad-hoc tutoring, not scheduled tutoring. 

Perhaps "Scheduled tutoring is limited in availability and reserved for students who instructors identify as likely to benefit from one-on-one time to solidify concepts. Students may also self-identify to be considered for needing this support."

CheezItMan commented on tutoring.md:
```diff
@@ -0,0 +1,57 @@
+### Which tutoring service should I use?
+Because of the differing availability of tutoring services we strongly advise that all students interested in tutoring first attend Open Tutoring or, if that is not possible, request dedicated Ad-hoc Tutoring through the mailing list.
+
+If both of those options are not available or are unsuccessful we suggest that you speak with one of your instructors and a more individual tutoring plan can be devised.
```
Maybe something about what they should do to prepare for a scheduled tutoring or open tutoring session, make a list of problems/questions etc.

CheezItMan reviewed (approved):
Looks pretty good.

sudocrystal reviewed (changes requested):
Looks really good, Charles!

Hamled committed:
Tutoring: Minor spelling and grammar corrections

Hamled committed:
Tutoring: Link to ada-tutors mailing list

Hamled committed:
Tutoring: Use email sending terminology

Instead of message posting terminology, which might imply that students
need to be part of the ada-tutors google group and use it like a forum.

Hamled committed:
Tutoring: Rephrase Scheduled Tutoring use case

It's less common that someone would be setup with Scheduled Tutoring
because they have missed in-class material, so we should not use that as
an example.

Hamled committed:
Tutoring: One-on-one instead of 1-on-1

Hamled committed:
Tutoring: Highlight differences between OT & AT

Open Tutoring is closer to extended TA hours, where-as Ad-hoc Tutoring
is one-on-one and allows for going more in-depth on a concept.

Hamled committed:
Tutoring: FAQ question about preparation

Hamled commented:
@kariabancroft @CheezItMan @sudocrystal I've updated this PR with changes relating to the feedback you provided.
